### ISSUE TITLE
Text colour contrast fails accessibility guidelines

### ISSUE THREAD

**Issue by gilgongo** [corrados/jamulus#65]
The current text colour used in the main Jamulus screen appears to be #8E918A for the text and #3F4346 for the background (I'm too incompetent to check that in the source though). 

![image](https://user-images.githubusercontent.com/4561747/79117994-87757b80-7d84-11ea-97f6-238a95b6746d.png)

This [fails WCAG AA](https://webaim.org/resources/contrastchecker/?fcolor=8E918A&bcolor=3F4346) for sufficient contrast at standard text size. 

[In order for it to pass](https://webaim.org/resources/contrastchecker/?fcolor=DCDCDA&bcolor=3F4346), suggest using #DCDCDA for the text instead.

**Comment by corrados** [comment]
It is #949494 actually. I changed it to #dcdcdc now.
